web-new/src/components/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import Link from "next/link";
+import Image from "next/image";
+import logoDarklake from "../../public/images/logo-h-darklake.png";
+
+const Header = () => {
+  return (
+    <header className="flex flex-row justify-between items-center pb-[64px]">
```

Comment:
We shouldn't use hardcoded values

Reply:
imo it is ok as long as not dynamically rendered and adding single values for each case can become a bigger mess than keeping these on records as constants 🙏